### PR TITLE
New scrapper->app sender, new models, refactoring

### PR DESCRIPTION
**Pull request by Devas**
Changed isbn to a single value in dto object, adjusted tests for it
Added objects for sending multiple Dtos to the app
Removed code smalls and bugs as shown in SonarCloud
Fixed BookModel toString test
Removed in-code address ip and took care of some code smells
Fixed a loading of ipaddress properties file
Added verbose log to the crawler app
Added progress when scrapping and sending books
Minor refactoring, removed two code smells
Changed to single line printing on console
Fixed printing progess on console: scrapping shows 100% now;

### CODE REVIEW

File: crawler/src/test/java/com/javaacademy/crawler/common/model/BookModelTest.java [hunk 1]
```diff
@@ -61,6 +62,18 @@ public void testEquals() {
 
     public void toStringTest() {
         BookModel bookModel = new BookModel();
-        assertEquals(bookModel.toString(), "BookModel(title=null, subtitle=null, authors=null, industryIdentifiers=null, categories=null, smallThumbnail=null, canonicalVolumeLink=null, saleability=null, listPriceAmount=0.0, listPriceCurrencyCode=null, retailPriceAmount=0.0, retailPriceCurrencyCode=null)");
+        assertEquals(bookModel.toString(),
+                "BookModel(industryIdentifier=null, subtitle=null, title=null, categories=null, authors=null, canonicalVolumeLink=null, saleability=null, smallThumbnail=null, listPriceAmount=0.0, retailPriceAmount=0.0, listPriceCurrencyCode=null, retailPriceCurrencyCode=null)");
+    }
+
+    @Test
+    public void testThis() {
```
Comment (Tymoteuszauke) [inline crawler/src/test/java/com/javaacademy/crawler/common/model/BookModelTest.java]: testThis? What does it test? I don't know.

Also, you don't have to add "Test" in every method name.

File: crawler/pom.xml [hunk 2]
```diff
@@ -30,6 +30,11 @@
             <artifactId>converter-gson</artifactId>
             <version>2.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>logging-interceptor</artifactId>
+            <version>3.3.0</version>
```
Comment (Dzionek95) [inline crawler/pom.xml]: it would be better to store versions in properties

File: crawler/src/main/java/com/javaacademy/crawler/App.java [hunk 3]
```diff
@@ -18,12 +23,18 @@
 
     public static void main(String[] args) {
```
Comment (Dzionek95) [inline crawler/src/main/java/com/javaacademy/crawler/App.java]: psvm?

File: crawler/src/main/java/com/javaacademy/crawler/common/booksender/BookSender.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+            }
+            sendBooksTo(serverIp, numberOfBooksSentAtOnce);
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                logger.log(Level.WARNING, "Interrupted waiting when sending books to book server: ", e);
+                Thread.currentThread().interrupt();
+            }
+            long progress = (booksToSend.values().stream().filter(aBoolean -> aBoolean).count()
+                    * 100) / booksToSend.size();
```
Comment (Dzionek95) [inline crawler/src/main/java/com/javaacademy/crawler/common/booksender/BookSender.java]: magic number

File: crawler/src/test/java/com/javaacademy/crawler/common/model/BookModelTest.java (hunk 1 above)
Comment (Dzionek95) [reply crawler/src/test/java/com/javaacademy/crawler/common/model/BookModelTest.java]: +1

**Pull request merged**